ehcache-impl/src/test/java/org/ehcache/impl/internal/store/heap/OnHeapStoreBulkMethodsTest.java

```diff
@@ -66,6 +76,24 @@ protected <Number, CharSequence> OnHeapStore<Number, CharSequence> newStore() {
         new NoopSizeOfEngine(), NullStoreEventDispatcher.nullStoreEventDispatcher(), new DefaultStatisticsService());
   }
 
+  @SuppressWarnings("unchecked")
+  protected <K, V> Store.Configuration<K, V> mockStoreConfig(int heapCount) {
```

Comment:
These methods are duplicating too much of the existing code. You should be able to have the existing methods call these ones to reduce the duplication.

Reply:
Removed new overloaded methods, Now using existing methods for my junits